tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
```

Comment:
Not sure if tox.ini should be put here, under root

Reply:
See above re: my thoughts on root-level files.